src/Logging/HTTPOutputHandler.php

```diff
@@ -70,6 +72,38 @@ public function setStatusCode($statusCode)
     }
 
     /**
+     * Set a formatter to use if Director::is_cli() is true
```

Comment:
Just `* @param $cliFormatter` eh?

 * The purpose of the method is defined in the method description
 * The type is in the type hint